common/src/constants.rs

```diff
@@ -3,4 +3,8 @@ pub const REGISTER_COUNT: u64 = 32;
 pub const REGISTER_START_ADDRESS: usize = 0;
 pub const RAM_START_ADDRESS: u64 = 0x80000000;
 pub const BYTES_PER_INSTRUCTION: usize = 4;
-pub const MEMORY_OPS_PER_INSTRUCTION: usize = 7;
\ No newline at end of file
+pub const MEMORY_OPS_PER_INSTRUCTION: usize = 8;
```

Comment:
Did I change this or did this get modified recently?

Reply:
This should be 7. 

4x byte addressable, rd, rs1, rs2.